lib/mix/lib/mix/task.ex

```diff
@@ -193,7 +193,10 @@ defmodule Mix.Task do
   end
 
   def alias?(task) when is_atom(task) do
-    Mix.Project.config[:aliases][task]
```

Comment:
Hm maybe, but wouldn't that be leaving the tests open to passing on truthy values?